staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -73,7 +73,7 @@ const (
 	APIServerIdentity featuregate.Feature = "APIServerIdentity"
 
 	// owner: @dashpole
-	// alpha: v1.22
+	// beta: v1.26
```

Comment:
```suggestion
	// beta: v1.27
```

LGTM  except this.

Reply:
yeah, also keep the alpha line so we can see the feature progress in file

Reply:
done, and done.